app/views/admin/enterprises/form/_add_new_unregistered_manager.html.haml

```diff
@@ -0,0 +1,22 @@
+%div#add_manager_modal
+  %form{ "data-reflex": "submit->InviteManager#invite", "data-reflex-serialize-form": true }
+    .margin-bottom-30.text-center
+      .text-big
+        = t('js.admin.modals.invite_title')
+
+    - if success
+      %p.alert-box.ok= t('user_invited', email: email)
+
+    - if error
+      %p.alert-box.error= error
+
+    = text_field_tag :email, nil, class: 'fullwidth margin-bottom-20'
+    = hidden_field_tag :enterprise_id, @enterprise&.id || enterprise.id
```

Comment:
Sorry if I missed it, but where could the class var `@enterprise` come from? I only see the local var `enterprise`.

Reply:
This comes from:

https://github.com/openfoodfoundation/openfoodnetwork/blob/22926b9dc489da8b121f1400b4d57c7d07117154/app/views/admin/enterprises/form/_users.html.haml#L92

which is a panel in enterprise:

https://github.com/openfoodfoundation/openfoodnetwork/blob/22926b9dc489da8b121f1400b4d57c7d07117154/app/views/admin/enterprises/_form.html.haml#L6

and then `@enterprise` is the current enterprise we want to edit.

Reply:
ah yes, of course 👍